projects/core/src/lib/colors/color-attribute-input/color-attribute-input.component.html

```diff
@@ -8,6 +8,7 @@
             [ngModel]="input?.key"
             (ngModelChange)="updateKey($event)"
         />
+        <mat-hint *ngIf="hasColorHint()" align="start">{{ colorHint(input?.key) }}</mat-hint>
```

Comment:
what I don't really understand: The user can change the value of the breakpoint here right? Then the label/hint would no longer correspond to the class right? Isn't that a problem?

Reply:
i believe in the power of data binding here. The model changes and then the hint will also change. At least it did when i tried it.